document-store/src/integrationTest/resources/mongo/updatable_collection_data.json

```diff
@@ -0,0 +1,175 @@
+[
+  {
+    "_id": 1,
+    "item": "Soap",
+    "price": 10,
+    "quantity": 2,
+    "date": "2014-03-01T08:00:00Z",
```

Comment:
As the query looks for `< 2022-08-09T18:53:17Z`, we have no document beyond it. So, can we add one document with data - `2023-08-09T18:53:17Z` for completeness?

Reply:
Done.